packages/babel-jest/package.json

```diff
@@ -16,6 +16,6 @@
     "babel-core": "^6.0.0"
   },
   "peerDependencies": {
-    "babel-core": "^6.0.0 || ^7.0.0-alpha || ^7.0.0"
```

Comment:
I think it's covered if we don't change the version

<img width="1232" alt="screen shot 2017-08-31 at 9 34 46 am" src="https://user-images.githubusercontent.com/588473/29925868-aebe2b34-8e2f-11e7-8987-068628686ce3.png">

but all good